src/types/symbol.rs

```diff
@@ -34,7 +34,7 @@ impl Hash for SymbolText {
         match self {
             SymbolText::Shared(text) => text.hash(state),
             SymbolText::Owned(text) => text.hash(state),
-            SymbolText::Unknown => 0.hash(state),
+            SymbolText::Unknown => "".hash(state),
```

Comment:
🗺️ The hashing behavior of `Symbol` now also uses the empty string if there's no text for agreement with `Borrow`.